src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -148,6 +148,10 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
         $__Command
         __cg2ExitCode=$?
 
+        if [ $__cg2ExitCode -ne 0 ]; then
```

Comment:
This is going to skip the code after `echo "Crossgen2 compilation finished, exit code"` that does not look desirable.